database/seeders/DatabaseSeeder.php

```diff
@@ -13,11 +14,49 @@ class DatabaseSeeder extends Seeder
      */
     public function run(): void
     {
-        // User::factory(10)->create();
-
-        User::factory()->create([
-            'name' => 'Test User',
-            'email' => 'test@example.com',
-        ]);
+        $users = [
+            [
+                'name' => 'Pasha',
+                'email' => 'www@PashkaPASS555777',
+                //         'password' => '555777',
+                'email_verified_at' => NULL,
```

Comment:
It doesn't matter when users verify their email. We should understand if the user's email is verified. Replace it with is_email_verified with a boolean value